theano/tests/test_flake8.py

```diff
@@ -30,8 +30,21 @@
     "tests/__init__.py",
     "compile/__init__.py",
     "compile/profiling.py",
+    "compile/sandbox/__init__.py",
+    "compile/tests/__init__.py",
+    "compile/tests/test_builders.py",
+    "compile/tests/test_misc.py",
+    "compile/tests/test_monitormode.py",
+    "compile/tests/test_function_module.py",
+    "compile/tests/test_shared.py",
+    "compile/tests/test_ops.py",
+    "compile/tests/test_pfunc.py",
+    "compile/tests/test_debugmode.py",
+    "compile/tests/test_profiling.py",
```

Comment:
It is OK to add empty `__init__.py` file in the whitelist, but here you are adding more.
Maybe these files got removed from the whitelist in the mean time, which is why you still had them when rebasing.
In any case, you should remove these `compile/tests/test_*.py` from the whitelist.